core/src/test/java/io/grpc/internal/ManagedChannelImplIdlenessTest.java

```diff
@@ -205,15 +204,14 @@ public void enterIdleModeAfterForceExit() throws Exception {
     newTransports.drainTo(transports);
     assertEquals(addressGroupList.size(), transports.size());
 
-    InterimTransport<ClientTransport> interimTransport = channel.tm.createInterimTransport();
+    channel.tm.createInterimTransport();
```

Comment:
I'm not sure I understand the question. The method returns a new interim transport. In this case, the test just needed to create it. I believe it is registered within `ManagedChannelImpl`, so it's not like we're just doing a `new` here to ignore the result.